technicalreport/introduction.tex

```diff
@@ -101,7 +101,9 @@ \subsubsection{Start-/Endpunkt auf der Fläche}
 
 \subsubsection{Routing bei zwei benachbarten Flächen}
 \label{target:Routing bei zwei benachbarten Flächen}
-Dieses Problem ist der Vollständigkeit halber aufgeführt, wird in dieser Arbeit aber nicht weiter verfolgt.
+
+Es soll geprüft werden, wie Fussgänger-Flächen bearbeitet werden können, die direkt benachbart sind. Wenn die beiden Flächen unabhängig voneinander verarbeitet werden, gibt es unter Umständen eine Kante, die von einem Fussgänger problemlos übergangen werden kann, von unserem Algorithmus aber als Grenze der Fläche wahrgenommen wird.
```

Comment:
> von unserem Algorithmus aber als Grenze der Fläche wahrgenommen wird

kann man das in der Anteilung schon sagen?

Reply:
Habs ein wenig umformuliert